superfluidDaoFront/src/components/CardYourDao.tsx

```diff
@@ -0,0 +1,10 @@
+const CardYourDao = (props: { name: string; description: string }) => {
```

Comment:
You can use destructuring here 
```suggestion
type Props = {
    name: string;
    description: string;
}

const CardYourDao = ({ name, description }: Props) => {
```